spark-submission-worker/src/main/java/org/apache/spark/k8s/operator/SparkClusterResourceSpec.java

```diff
@@ -129,6 +131,7 @@ private static StatefulSet buildMasterStatefulSet(
         .addToLabels(LABEL_SPARK_ROLE_NAME, LABEL_SPARK_ROLE_MASTER_VALUE)
         .endMetadata()
         .withNewSpec()
+        .withSchedulerName(scheduler)
```

Comment:
Although we can set `schedulerName` like this, we cannot add a test case because `fabric8` doesn't have `getSchedulerName` unlike `PodSpec`.
- https://github.com/fabric8io/kubernetes-client/blob/main/kubernetes-model-generator/kubernetes-model-apps/src/generated/java/io/fabric8/kubernetes/api/model/apps/StatefulSetSpec.java